frontend/src/components/AppContainer/index.tsx

```diff
@@ -21,9 +19,6 @@ const AsyncCollection = loadable<Props>(
   () => /*webpackChunkName: 'AsyncCollection' */ import("src/views/Collection")
 );
 
-// (thuang): Check for Create Collection feature flag
-get(FEATURES.CREATE_COLLECTION);
```

Comment:
For modularization, `get(FEATURES.CREATE_COLLECTION)` is used at the callsite (components that care about this flag), so we don't need centralized management for feature flags, which causes problems such as not knowing if deleting this call will break any components